docs/changelog/91188.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 91188
```

Comment:
I believe we don't need a PR .yaml changelog for doc changes. Can you remove this file please?

Reply:
Removed. Thank you.